src/lib/assets/skeletons/SkeletonInlineBlock.tsx

```diff
@@ -0,0 +1,14 @@
+import React from 'react'
+import Skeleton, { SkeletonProps } from '@material-ui/lab/Skeleton'
+
+/**
+ * Skeleton with a display value of `inline-block`. MUI applies `display: block` with high specificity, so it's easiest to just apply the style to the component.
+ */
+export const SkeletonInlineBlock: React.FC<SkeletonProps> = props => {
+  return (
+    <Skeleton
+      style={{ ...props.style, display: 'inline-block' }}
```

Comment:
I could write a super specific CSS class for the sake of styling one component, do inline styling (as shown in code), or add a simple CSS class like
```css
.SkeletonButtonInline {
	display: inline-block !important;
}
```

I prefer inline styling since this style is necessarily tightly coupled with the component, but not sure if we have another existing pattern.

Reply:
I think an inline style is the cleanest approach for this case (prefer over the !important directive)